charts/sonarqube/templates/secret.yaml

```diff
@@ -26,3 +26,21 @@ metadata:
 type: Opaque
 data:
   SONAR_WEB_SYSTEMPASSCODE: {{ .Values.monitoringPasscode | b64enc | quote }}
+---
+{{- if .Values.account }}
+{{- if .Values.account.adminPassword }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "sonarqube.fullname" . }}-admin-password
+  labels:
+    app: {{ template "sonarqube.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+type: Opaque
+stringData:
+  password: {{ .Values.account.adminPassword | urlquery }}
```

Comment:
The only real difference is the new admin password doesn't have a default since `change-admin-password-hook.yml` is only applied if the new admin password is defined.